.github/workflows/golangci-lint.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
-      - name: Set up Go 1.18
+      - name: Set up Go 1.19
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18.10
+          go-version: 1.19
```

Comment:
Are we now using `1.19` across the full network stack? It's very nice to have that consistency for validators.

Reply:
we are using 1.19 across the stack.